docs/pages/workflow/continuous-native-generation.mdx

```diff
@@ -25,91 +26,254 @@ Expo as a framework enables **Continuous Native Generation** by combining the fo
-- [Cross-platform End-2-End testing](https://github.com/expo/config-plugins/tree/main/packages/detox) &mdash; Configure native apps to support E2E testing with Detox in a single-line.
-- [Entire Firebase suite](https://rnfirebase.io/) &mdash; Here you can see the entire native Firebase suite going from a multi-step native process across multiple IDEs, down to basic JSON configuration.
-- [Cross-platform Home screen Widgets](https://github.com/gaishimo/eas-widget-example) &mdash; This Expo Config Plugin can generate a home screen widget for Android and iOS.
-- [Notification extension and code signing](https://github.com/OneSignal/onesignal-expo-plugin) &mdash; This Expo Config Plugin not only generates a notification extension target on iOS. However, it also augments the EAS credentials service to keep zero-config code signing working.
-- [Apple App Clips](https://github.com/bndkt/react-native-app-clip) &mdash; This Expo Config Plugin takes the process of generating an Apple App Clip from a multi-step process, ranging across multiple targets, and shakes it down to a single line `["react-native-app-clip", { "name": "My App Clip" }]`.
+- **No native code or configuration side-effects**: Libraries without native code or configuration side-effects, such as `react-native-blurhash`, can seamlessly integrate with `npx expo prebuild`. They can rely on Node Module Resolution and Expo Autolinking to link into the native project without requiring any additional configuration.
 
-At any point, these features can be easily added, and just as easily removed, without any side effects. CNG allows developers to experiment with complex features, and iterate on them quickly without worrying about the long-term maintenance costs or potential orphaned code in their project or uncommitted native caches.
+- **Native code with no additional setup after install**: Libraries with native code can often be installed and linked automatically with [Expo Autolinking](/more/glossary-of-terms#autolinking) which runs before the native app is built.
 
-This nets out to better apps, built faster.
+- **Additional configuration side-effects and setup**: Libraries that require additional configuration side-effects can adopt CNG by creating [Expo Config Plugins](/config-plugins/introduction/) for their libraries. This approach enables library authors to automate adding values such as permission messages to the **Info.plist**, or injecting targets in the Xcode project.
 
-## Frequently Asked Questions
+- **Libraries Dependent on Native Runtime Hooks**: Libraries that depend on specific native runtime hooks, such as intercepting the initial launch URL via the `AppDelegate`, `MainActivity`, `MainApplication`, and so on, can utilize [**Lifecycle listeners**](/modules/android-lifecycle-listeners/) to minimize the side-effects of their library.
```

Comment:
```suggestion
- **Libraries Dependent on Native Runtime Hooks**: Libraries that depend on specific native runtime hooks, such as intercepting the initial launch URL via the `AppDelegate`, `MainActivity`, `MainApplication`, and so on, can utilize [**Lifecycle listeners**](/modules/android-lifecycle-listeners/) in the Expo Modules API. These lifecycle listeners allow these runtime hooks to be applied via Expo Autolinking instead of by modifying these standard native project files, eliminating the need for a config plugin.
```